api/factory/poc.ts

```diff
@@ -0,0 +1,27 @@
+import { ethers } from "ethers";
+import { OWNER_PRIVATE_KEY } from "../config";
+import { FactoryAPI } from "./factory";
+
+const jsonProvider = new ethers.providers.JsonRpcProvider(
+  "http://127.0.0.1:8545"
+);
+
+const wallet = new ethers.Wallet(OWNER_PRIVATE_KEY, jsonProvider);
+
+const contractAddress = "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512";
+
+const factory = new FactoryAPI(contractAddress, wallet);
+
+const MADE_UP_ADDRESS = "0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d";
```

Comment:
use usecase specific names - everything is made up, but this serves some purpose down in the test
is it operator? is it factor? is it vault?